ui/main/src/app/services/notifications/NotificationDecision.ts

```
@@ -40,7 +42,18 @@ export class NotificationDecision {
-        NotificationDecision.cleanSentCards();
+
+        // Avoid launching the regular cleanSentCards more than once
+        // this is to avoid having multiple intervals running in parallel
+        // in unit tests.In production, this is not a problem as the
+        // init is only called once
+        if (NotificationDecision.cleanTask) {
+            clearInterval(NotificationDecision.cleanTask);
+        }
+
+        NotificationDecision.cleanTask = setInterval(() => {
+            this.cleanSentCards();
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)